### ISSUE TITLE
dap-php-setup fails to unzip

### ISSUE THREAD

**Issue by hacktivista** [emacs-lsp/dap-mode#598]
When I run `dap-php-setup` I get the following output on `*Messages*`

```
Contacting host: marketplace.visualstudio.com:443
uncompressing publicsuffix.txt.gz...done
Wrote /tmp/extJNNjZz.zip
[/tmp/extJNNjZz.zip]
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of /tmp/extJNNjZz.zip or
        /tmp/extJNNjZz.zip.zip, and cannot find /tmp/extJNNjZz.zip.ZIP, period.
dap-php: Downloading done!
```

**Comment by bridgesense** [comment]
Yeah, this is a real deal-breaker for new ones checking out dap-php for the first time.  dap-php-setup only works about 5% of the time.  I had to run it several times.  

It's a shame really, because this is such a great IDE for PHP.

I keep a copy of my .extension directory backed up elsewhere.  Someone new rolling in isn't going to know anything about that.

**Comment by dustinpaluch** [comment]
@bridgesense I'm trying to manually get around this problem. What exactly am I supposed to download?

**Comment by tomterl** [comment]
You can download the vsix Release from xdebug/vscode-php-plugin* and extract it (it's a zip-file) in the place where your setup expects it, in my case inside `.emacs.d/.local/cache/.extension/vscode/xdebug.php-debug`.

*: https://github.com/xdebug/vscode-php-debug/releases
